libgnucash/app-utils/gnc-optiondb.cpp

```diff
@@ -1194,13 +1204,20 @@ gnc_option_db_book_options(GncOptionDB* odb)
     gnc_register_string_option(odb, business_section, N_("Company ID"), "c5",
                                N_("The ID for your company (eg 'Tax-ID: 00-000000)."),
                                empty_string);
-
+    gnc_register_invoice_print_report_option(odb, business_section,
+                                 OPTION_NAME_DEFAULT_INVOICE_REPORT, "e1",
+                                 N_("The invoice report to be used for printing."),
+                                 empty_string);
```

Comment:
That last value is the default value. You want it to be the guid-string for the Printable Invoice Report, not empty. Ensuring that there's a correct default value means that you don't have to handle setting a default in code anywhere.

Reply:
I left it as `empty_string` as I did not want to change it to the guid value of `PRINTABLE_INVOICE_GUID`. Instead I change this in `business-options-gnome.cpp` where I can also use the define.